sklearn/preprocessing/label.py

```diff
@@ -433,12 +440,56 @@ def label_binarize(y, classes, neg_label=0, pos_label=1, sparse_output=False):
 
     Binary targets transform to a column vector
 
+    >>> label_binarize([0, 1, 1, 0], neg_label=-2, pos_label=0)
+    array([[-2],
+           [ 0],
+           [ 0],
+           [-2]])
+
+    >>> label_binarize([1, 0, 0, 1], neg_label=1, pos_label=-2)
+    array([[-2],
+           [ 1],
+           [ 1],
+           [-2]])
+
     >>> label_binarize(['yes', 'no', 'no', 'yes'], classes=['no', 'yes'])
     array([[1],
            [0],
            [0],
            [1]])
 
+    Binary with example for positive and negative labels
+
+    >>> label_binarize(['yes', 'no', 'no', 'yes'], classes=['no', 'yes'],
+    ...     neg_label=-1, pos_label=2)
+    array([[ 2],
+           [-1],
+           [-1],
+           [ 2]])
+
+    >>> label_binarize(['yes', 'no', 'no', 'yes'], classes=['no', 'yes'],
```

Comment:
I would discard it. We do not use docstrings as unit tests. We use them as examples in context like this one. Besides `.toarray()` is a very weak test of what is output.